plugin/registry.go

```diff
@@ -129,3 +129,8 @@ func (r *stubRoot) ChildSchemas() []*EntrySchema {
 func (r *stubRoot) List(context.Context) ([]Entry, error) {
 	return []Entry{}, nil
 }
+
+const registryDescription = `
+Welcome to Wash, a UNIX-like shell that lets you manage all your things (entries) as
```

Comment:
I'm undecided on using `thing` all over the place. I think it's ok to use entry consistently, and maybe describe what that means in one place.

Reply:
Updated.